Tests/GRPCProtobufCodeGenTests/ProtobufCodeGeneratorTests.swift

```diff
@@ -58,13 +58,13 @@ final class ProtobufCodeGeneratorTests: XCTestCase {
         import DifferentModule
         import ExtraModule
 
-        internal enum Helloworld_Greeter {
+        internal enum HelloWorld_Greeter {
```

Comment:
Don't we want the name to be `Hello_World_Greeter` here? I.e. the `Hello_World` prefix should match that used by the messages (`Hello_World_HelloRequest` and `Hello_World_HelloReply`).